service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -205,8 +206,21 @@ public ResponseEntity<PostMethodResponse> postMethod(PostMethodRequest postMetho
         String organization = githubUrlComponents.org();
         branchOrTagName = githubUrlComponents.branchOrTag();
 
-        String githubToken = ecmService.getAccessToken();
-        Boolean isPrivate = gitHubService.isRepoPrivate(organization, repository, githubToken);
+        String githubToken;
+        Boolean isPrivate;
+
+        try {
```

Comment:
nit: since this whole logic is related to determining if the repo is private, I wonder if it can be moved out of Controller and into the Github service layer itself. But if this is more effort and considering this is a time sensitive fix I am happy for it to be punted.